docs/content/labs/common/watching/watching-cp.md

```diff
@@ -0,0 +1,20 @@
+Now it's a matter of waiting for the cluster to finish the deployment, so let's take a look at some useful commands on the Management cluster side:
+
+```bash
+export KUBECONFIG=/root/.kcli/clusters/hub-ipv4/auth/kubeconfig
+
+watch "oc get pod -n hypershift;echo;echo;oc get pod -n clusters-hosted-ipv4;echo;echo;oc get bmh -A;echo;echo;oc get agent -A;echo;echo;oc get infraenv -A;echo;echo;oc get hostedcluster -A;echo;echo;oc get nodepool -A;echo;echo;"
```

Comment:
@jparrill I noticed that is command includes `ipv4`. Will it work for IPv6 and dual stack, too?

Reply:
Yes, `clusters-hosted-ipv4` is the namespace's name for the HostedControlPlane, it could be `clusters-hosted` or `clusters-sample`, when `clusters` is the HostedCluster's namespace and `sample` is the HostedCluster's name.